src/main/java/io/sentry/samples/example/Application.java

```diff
@@ -32,53 +33,43 @@
 	private static Map<String, Integer> inventory = new HashMap<>();
 
 	private void checkout(List<Item> cart) {
+
 		Map<String, Integer> tempInventory = inventory;
 		for(Item item : cart) {
 			int currentInventory = tempInventory.get(item.getId());
 			if(currentInventory <= 0) {
 				String message = "No inventory for " + item.getId();
+				System.out.println(message);
 				throw new RuntimeException(message);
 			}
-
-			tempInventory.put(item.getId(), currentInventory--);
+			currentInventory = currentInventory - 1;
+			tempInventory.put(item.getId(), currentInventory);
 		}
 		inventory = tempInventory;
 	}
 
-	// Class constructor to initialize logging 
 	public void Application() {
 
 	}
 
+	@CrossOrigin(origins = "http://localhost:5000")
```

Comment:
not sure why other people didn't get CORS problems when connecting React<>Sping, but I did, so added this to fix it.